src/langs/zig/wheel-part.zig

```diff
@@ -0,0 +1,9 @@
+extern fn jsrandom() callconv(.C) f64;
```

Comment:
I use this approach for some of the other languages as well (because their native way for generating random numbers does not compile to wasm). I am not familiar with Zig, but I was wondering if there was a native way to generate random numbers in Zig that we could use here

Reply:
Done!